repos.yaml

```diff
@@ -961,6 +961,43 @@
     - desc: Package details
       url: 'https://software.opensuse.org/package/{name}'
   tags: [ all, production, opensuse, rpm ]
+  #
```

Comment:
Please fix, also please move the block before tumbleweed to make it consistent with the freshness and sorting on the site.